src/wallet/wallet.h

```diff
@@ -49,6 +49,14 @@ std::vector<std::shared_ptr<CWallet>> GetWallets();
 std::shared_ptr<CWallet> GetWallet(const std::string& name);
 std::shared_ptr<CWallet> LoadWallet(interfaces::Chain& chain, const WalletLocation& location, std::string& error, std::string& warning);
 
+enum WalletCreationStatus {
```

Comment:
`enum class`

Reply:
I think I'll leave it as is to avoid invalidating existing ACKs.